apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  .form .formrow{
+    text-align: left;
+    margin:25px auto;
+    max-width: 400px;
+  }
+  
```

Comment:
@SakinaBoriwala21 Is there a reason we have left more than 1 line here. I don't think it is required. Please remove extra lines.

Reply:
No specific reason for the extra line. Yeah, I will do that.